app/web/src/AppRoutes.tsx

```diff
@@ -68,6 +42,65 @@ import {
   userRoute,
 } from "./routes";
 
+const EditCommunityInfoPage = React.lazy(
```

Comment:
We might have to do more ops magic here to ensure we don't flush the cache of old JS chunks now that we started code splitting, otherwise we might start [running into this `ChunkLoadError` issue described here](https://dimaip.github.io/2020/04/25/deploying-apps-with-code-splitting/)

Reply:
Oh damn